controllers/comments-controllers.js

```diff
@@ -15,3 +18,18 @@ exports.getCommentsByArticleId = async (req, res, next) => {
     next(err);
   }
 };
+
+exports.postComment = async (req, res, next) => {
+  try {
+    const { article_id: id } = req.params;
+    const { username, body } = req.body;
+
+    const {
```

Comment:
Nice use of nested array destructuring! Might argue that the model should do this and provide the controller with nothing more than it needs - but as long as you're consistent this isn't a problem!